internal/lib/control_config.go

```diff
@@ -244,6 +263,18 @@ func (c *akoControlConfig) SetCRDEnabledParams(cs akocrd.Interface) {
 	}
 }
 
+func (c *akoControlConfig) Setv1alpha2CRDEnabledParams(cs v1alpha2akocrd.Interface) {
+	timeout := int64(120)
+	_, oauthSamlErr := cs.AkoV1alpha2().OAuthSamlConfigs(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
+	if oauthSamlErr != nil {
+		utils.AviLog.Infof("ako.vmware.com/v1alpha2/OAuthSamlConfig not found/enabled on cluster: %v", oauthSamlErr)
+		c.oauthSamlConfigEnabled = false
+	} else {
+		utils.AviLog.Infof("ako.vmware.com/v1alpha2/OAuthSamlConfig enabled on cluster")
+		c.oauthSamlConfigEnabled = true
```

Comment:
We have to honor Akshay's new changes related to the CRD enable flag once his changes are merged into the master.

Reply:
Sure, I have noted this.